src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -302,6 +306,9 @@
   <Import Project="..\..\build\native.targets" />
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
+    <Import Project="..\..\packages\Microsoft.Build.Tasks.Git.1.0.0-beta2-19351-01\build\Microsoft.Build.Tasks.Git.targets" Condition="Exists('..\..\packages\Microsoft.Build.Tasks.Git.1.0.0-beta2-19351-01\build\Microsoft.Build.Tasks.Git.targets')" />
```

Comment:
Would it be better to put these in the RepoRoot artifacts directory? I know we have been pushing for that across all projects.

Reply:
These projects aren't going to work w/ `<PackageReference/>` and therefore don't know much about the more advanced (and recent) configuration options. But, I'm not done yet since these projects are also not auto-restoring ☹️